runners/spark/src/main/java/org/apache/beam/runners/spark/translation/TranslationUtils.java

```diff
@@ -270,9 +322,73 @@ public static void rejectStateAndTimers(DoFn<?, ?> doFn) {
 
   public static <T> VoidFunction<T> emptyVoidFunction() {
     return new VoidFunction<T>() {
-      @Override public void call(T t) throws Exception {
+      @Override
+      public void call(T t) throws Exception {
         // Empty implementation.
       }
     };
   }
+
+  public static <T, K, V> PairFlatMapFunction<Iterator<T>, K, V> pairFunctionToPairFlatMapFunction(
```

Comment:
This is a v. useful public function, we should probably have a Javadoc for it.
Same comment for `functionToFlatMapFunction`